api/v1alpha1/environment_types.go

```diff
@@ -113,6 +113,10 @@ type KubernetesClusterCredentials struct {
 	// APIURL is a reference to a cluster API url defined within the kube config file of the cluster credentials secret.
 	APIURL string `json:"apiURL"`
 
+	// IngressDomain is the cluster domain.
+	// For example, in minikube it would be $(minikube ip).nip.io and in OCP it would look like apps.xyz.rhcloud.com
```

Comment:
Should we say it's required when running on non-OCP clusters?

Reply:
yeah, make a mention of webhook validations